### PR TITLE
Add tuning section for Omni-Path devices

### PR DESCRIPTION
**Pull request by aravindksg**
- Add overall FAQ entry in Omni-Path
- Per Jeff's request in Issue https://github.com/open-mpi/ompi/issues/6256
  add note on Multi-rail and Multi-HFI support in PSM2.

Signed-off-by: Aravind Gopalakrishnan <Aravind.Gopalakrishnan@intel.com>

### CODE REVIEW

**Comment by aravindksg** [comment]
@matcabral @jsquyres

**Comment by matcabral** [comment]
@jsquyres please add us are reviewers (me no permissions here). thanks

File: faq/running.inc [hunk 1]
```diff
@@ -1663,6 +1663,19 @@ For more information refer to
...
+
+$anchor[] = "opa-support";
+
+$opa = "<a href\"https://www.intel.com/content/www/us/en/support/products/92003/network-and-i-o/fabric-products/intel-omnipath-host-fabric-interface-products.html\">";
+
+$a[] = "Currently, Open MPI supports PSM2 MTL and OFI MTL components which can
+be used to communicate with Intel ${opa}Omni-Path</a> (OPA) software stack
+
+For guidlines on tuning run-time characteristics when using OPA devices, please
+refer to <a href=\"?category=opa\">this FAQ entry</a>.";
```
[the remark this answers is not in the document]
Comment (aravindksg) [reply faq/running.inc]: I think "<a href=\"?category=opa\"> should resolve correctly here. (Looking at the FAW entry on "What kind of CUDA support exists in Open MPI", I see it is similar when pointing to "buildcuda" and "runcuda" entries - https://github.com/open-mpi/ompi-www/blob/master/faq/running.inc#L1612).

File: faq/opa.inc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+<li> PSM2_MULTIRAIL_MAP=unit:port,unit:port... </li>
+
+</ul>
+
+The variables above may be included in the mpirun command line or in the
+environment. Example:
+
+<geshi bash>
+shell$ mpirun -mca mtl [psm2|ofi] -x PSM2_MULTIRAIL=1 -np 2 -H host1,host2 ./a.out
```
Comment (matcabral) [inline faq/opa.inc]: may want to add a note saying that when OFI is used the PSM2 provider must be used.  Maybe also add a link to the ""How can Open MPI communicate with Intel Omni-Path Architecture (OPA)" section

File: faq/running.inc [hunk 3]
```diff
@@ -1663,6 +1663,19 @@ For more information refer to
...
 ";
 
+$q[] = "How can Open MPI communicate with Intel Omni-Path Architecture (OPA)
+based devices?";
+
+$anchor[] = "opa-support";
+
+$opa = "<a href\"https://www.intel.com/content/www/us/en/support/products/92003/network-and-i-o/fabric-products/intel-omnipath-host-fabric-interface-products.html\">";
+
+$a[] = "Currently, Open MPI supports PSM2 MTL and OFI MTL components which can
```
Comment (matcabral) [inline faq/running.inc]: minor comment:  ...  OFI MTL (using PSM2 OFI provider) ...

File: faq/opa.inc (hunk 2 above)
Comment (aravindksg) [reply faq/opa.inc]: Fixed this in the new commit

File: faq/running.inc (hunk 3 above)
Comment (aravindksg) [reply faq/running.inc]: Fixed in new commit

**Pull request merged**